src/main.cpp

```diff
@@ -100,17 +103,44 @@ namespace
         } else if (version_str == "v4.3") {
             return ebi::vcf::Version::v43;
         }
-        
+
         throw std::invalid_argument{"Please choose one of the accepted VCF fileformat versions"};
     }
-    
-    
-    
+
+    std::vector<std::shared_ptr<ebi::vcf::ReportWriter>> get_outputs(std::string const &output_str, std::string const &input) {
+        std::vector<std::string> outs;
+        ebi::util::string_split(output_str, ",", outs);
+        size_t initial_size = outs.size();
+
+        // don't write several times to the same output
+        std::sort(outs.begin(), outs.end());
+        std::unique(outs.begin(), outs.end());
+        if (initial_size != outs.size()) {
+            std::cerr << "Warning, duplicated outputs! will write just once to each output specified by -o/--output" << std::endl;
+        }
+
+        std::vector<std::shared_ptr<ebi::vcf::ReportWriter>> outputs;
+
+        for (auto out : outs) {
+            if (out == "sqlite") {
+                outputs.push_back(std::make_shared<ebi::vcf::SqliteReportWriter>(input + ".errors.db"));
+            } else if (out == "stdout") {
+                outputs.push_back(std::make_shared<ebi::vcf::StdoutReportWriter>());
+            } else if (out == "silent") {
+                ; // do nothing, leaving an empty vector if necessary
+            } else {
+                throw std::invalid_argument{"Please use only valid outputs"};
```

Comment:
Use the same message structure as in the rest of errors: "Please choose one of the accepted xxx"